__tests__/app.test.js

```diff
@@ -193,4 +203,70 @@ describe("app()", () => {
         });
     });
   });
+
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("POST 201: /api/articles/:article_id/comments should return a 201 status code", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "I love this article",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then((data) => {
+          const { body } = data;
+          const comment = body.comments;
+          expect(comment).toMatchObject({
+            comment_id: 19,
+            body: "I love this article",
+            article_id: 3,
+            author: "butter_bridge",
+            votes: 0,
+            created_at: expect.any(String),
+          });
+        });
+    });
+
+    test("POST 400: /api/articles/:article_id/comments should return a 400 status code 'missing required field' if missing required fields", () => {
+      const newComment = {
+        body: "I love this article",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("missing required field");
+        });
+    });
+
+    test("POST 400: /api/articles/:article_id/comments should return a 400 'bad request' if bad request made", () => {
```

Comment:
Instead of saying 'if bad request made' you could tell what it is, like 'if the id has the wrong data type'